packs/bit-bucket/actions/list_ssh_keys.yaml

```diff
@@ -0,0 +1,10 @@
+name: list_ssh_keys
+runner_type: run-python
+description: List all SSH keys for a user
+enabled: true
+entry_point: list_ssh_keys.py
+parameters:
+  repo:
```

Comment:
Hm, this action performs operation on an user account and not on a repository. Does it still need the repo argument?